Makefile

```diff
@@ -29,7 +29,7 @@ TIMEOUT = 60
 .DEFAULT_GOAL := precommit
 
 .PHONY: precommit ci
-precommit: dependabot-generate license-check misspell go-mod-tidy vanity-import-fix golangci-lint-fix test-default
+precommit: dependabot-generate license-check misspell go-mod-tidy vanity-import-fix generate golangci-lint-fix test-default
```

Comment:
I would prefer to run `go generate` before the linters. But this would change the existing behavior on CI. See (`build` depends on `generate`).: https://github.com/open-telemetry/opentelemetry-go-contrib/blob/5da82d19baf8dbba06651c9118df778411849d0d/.github/workflows/ci.yml#L48-L49

I prefer to address it in a separate PR.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3819

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3820